test/parallel/test-fs-utils-get-dirents.js

```diff
@@ -65,6 +65,31 @@ const filename = 'foo';
     },
     ));
 }
+{
+  // Intentional error in lstat
+  let path = '';
+  if (common.isWindows) {
+    path = 'c:\\dev\\null\\does\\not\\exist';
+  } else {
+    path = '/dev/null/does/not/exist';
+  }
+
+  const errmsg_enotdir_start =
+    'ENOTDIR: not a directory, ';
+  const errmsg_enoent_start =
+    'ENOENT: no such file or directory, ';
```

Comment:
1. You can probably store these outside the brackets

2. IMO you really only need to match the error code

Reply:
Applied option 2, and removed that lines as it became unnecessary